docs/source/docformat/epytext.rst

```diff
@@ -17,7 +17,7 @@ All markup should work except the indexed terms ``X{}`` tag, which has been remo
 Fields
 ------
 
-See `fields section <codedoc.rst#fields>`__.
+See `fields section <../codedoc.html#fields>`__.
```

Comment:
Looks good. Thanks.

But maybe is best to add a dedicated Sphinx directive to reference this via https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role

```suggestion
See :ref:`fields section <codedoc-fields>`.
```
and add this inside codedoc.rst before the fields header.

```
.. _codedoc-fields:
```

In this way, even if we move the section, the links are automatically updated.

Reply:
Thanks!